app/views/cart_items/show.html.erb

```diff
@@ -0,0 +1,21 @@
+<h1> Cart Contents: </h1>
+<table class="bordered striped highlight centered responsive-table">
+  <thead>
+    <tr>
+      <th data-field="Image">Image</th>
+      <th data-field="Title">Title</th>
+      <th data-field="Description">Description</th>
+      <th data-field="Travesty">Travesty</th>
+      <th data-field="Price">Price ($)</th>
+      <th data-field="Quantity">Quantity</th>
+      <th data-field="LineTotal">Line Total</th>
+      <th data-field="Remove Item">Remove Item</th>
+    </th>
+  </tr>
+</thead>
+<tbody>
+  <%= render partial: "cart_items", collection: @cart.items, as: :item %>
+</tbody>
+</table>
+
+<h2><%= "Total Price: $#{@cart.total_price}.00" %></h2>
```

Comment:
Could use this here:

<%= "Total Price: " %><%= number_to_currency(@cart.total_price) %>